jujugui/static/gui/src/app/components/deployment-flow/_deployment-flow.scss

```diff
@@ -23,27 +23,210 @@
         padding-top: 40px;
     }
 
+    &__section {
+        margin-bottom: 40px;
+        padding: 20px;
+        background-color: $light-grey;
+
+        &--clear {
+            padding: 0;
+            background-color: transparent;
+        }
+
+        &-title {
+            .svg-icon {
+                margin-right: 10px;
+                vertical-align: text-top;
+            }
+
+            &-aside {
+                padding-left: 20px;
+                font-size: 16px;
+            }
+        }
+    }
+
+    &__top-action {
+        float: right;
+    }
+
+    &__label {
+        margin-bottom: 10px;
+        color: $warm-grey;
+        font-size: 0.688em;
+    }
+
+    &__row-header {
+        font-weight: 400;
+    }
+
+    &__row-header,
+    &__row {
+        border-bottom: 1px solid $light-mid-grey;
+    }
+
+    .button--neutral {
+        background-color: $white;
+    }
+
     &__clouds {
+        &-list {
+            @extend %list-style;
+        }
 
+        &-cloud {
+            background-color: $white;
+            box-shadow: $box-shadow;
+            text-align: center;
+            cursor: pointer;
+            margin-top: 20px;
+            margin-bottom: 0;
+
+            &:hover {
+                background-color: $light-grey;
+            }
+
+            &-logo {
+                display: block;
+                line-height: 120px;
```

Comment:
I think this would be nicer assigned to a descriptive variable such as:
```sass
$logo-height: 120px;
display: block;
line-height: $logo-height;
```